src/System/Hapistrano/Maintenance.hs

```diff
@@ -0,0 +1,28 @@
+{-# LANGUAGE TemplateHaskell   #-}
+module System.Hapistrano.Maintenance (
+  writeMaintenanceFile,
+  deleteMaintenanceFile
+) where
+
+import Path ( mkAbsDir, mkRelFile, parseRelDir, Path, Abs, Rel, Dir, File, (</>))
+import System.Hapistrano.Types
+import System.Hapistrano.Core
+import System.Hapistrano.Commands
+
+writeMaintenanceFile :: Path Abs Dir -> Path Rel Dir -> Path Rel File -> Hapistrano()
+writeMaintenanceFile deployPath relDir fileName =
+  let foo = deployPath </> relDir
+      fullpath = relDir </> fileName
+      root = deployPath </> fullpath
+  in do
+    exec(MkDir foo) Nothing
+    exec(Touch root) Nothing
+    exec (BasicWrite root "<!DOCTYPE html><html><head><title>Maintenance</title><style type=\"text/css\">body {width: 400px;margin: 100px auto;font: 300 120% \"OpenSans\", \"Helvetica Neue\", \"Helvetica\", Arial, Verdana, sans-serif;}h1 {font-weight: 300;}</style></head><body><h1>Maintenance</h1><p>The system is down for maintenance</p><p>It'll be back shortly</p></body></html>") Nothing
```

Comment:
@FranzGB It is possible to write this HTML content in a separated variable and as multiple lines:
```hs
htmlContent :: String
htmlContent = "\
  \<!DOCTYPE html>\
  \<html>\
    \<head>\
    \</head>\
    ...
  \<html>\
  \"
```
then:
```suggestion
    exec (BasicWrite root htmlContent) Nothing
```
Feel free to rename `htmlContent` with anything you prefer. Thanks!